src/plugins/dbus/dbus.c

```diff
@@ -90,31 +82,15 @@ int elektraDbusCommit (Plugin * handle, KeySet * returned, Key * parentKey)
 	ElektraDbusPluginData * pluginData = elektraPluginGetData (handle);
 	ELEKTRA_NOT_NULL (pluginData);
 
-	// because elektraLogchangeGet will always be executed before elektraLogchangeCommit
-	// we know that oldKeys must exist here!
-	KeySet * oldKeys = pluginData->keys;
+	KDB * kdb = elektraPluginGetKdb (handle);
+	const ChangeTrackingContext * context = elektraChangeTrackingGetContextFromKdb (kdb);
```

Comment:
I don't think we should offer a way to get `kdb` (unless there is a valid use case for this). Better to have a getter directly for getting the change tracking.